docs/index.js

```diff
@@ -0,0 +1,15 @@
+document.querySelector('#Book').addEventListener('click', function() {
```

Comment:
HI lmk if this works 

`let modalMask = document.querySelector('#modal-mask')`
outside of the functions declared once and the rest of the functions remove the let like this 

`   document.querySelector('#modal-close').addEventListener('click', function() {
    modalMask.style.display= 'none'
   })`

Reply:
format the function better tho